File: service/SurfWebAutofac/Model/Dtos/Players/PlayerFailDto.cs
﻿namespace Model.Dtos.Players;

public class PlayerFailDto
{
    /// <summary>
    ///     地图id
    /// </summary>
    public long MapId { get; set; }

    /// <summary>
    ///     地图名称
    /// </summary>
    public string MapName { get; set; }

    /// <summary>
    ///     难度
    /// </summary>
    public string Difficulty { get; set; }
    /// <summary>
    /// 图片
    /// </summary>
    public string Img { get; set; }

    /// <summary>
    ///     阶段
    /// </summary>
    public List<int> Stages { get; set; } = new();
}

File: service/Java/SurfWeb/SurfWeb/system/src/main/java/com/surfweb/system/dto/players/PlayerFailDto.java
package com.surfweb.system.dto.players;

import lombok.Data;

import java.util.List;

@Data
public class PlayerFailDto {

    /**
     * 地图id
     */
    private long mapId;

    /**
     * 地图名称
     */
    private String mapName;

    /**
     * 难度
     */
    private String difficulty;

    /**
     * 图片
     */
    private String img;

    /**
     * 阶段
     */
    private List<Integer> stages;
}
